kaptan/__init__.py

```diff
@@ -52,20 +53,39 @@ def upsert(self, key, value):
         self.configuration_data.update({key: value})
         return self
 
+    def _is_python_file(self, value):
```

Comment:
Regarding that refactor that idea, probably out of scope for this PR.  I'll make a separate issue for that to tackle at a later time.